submissions/Adebiyi_Osinibi/HTML/reg_page.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+<head>	
+	<title>Signup | Tropica! Home of Quality African Products </title>
+	<link href = "stylereg.css" rel = "stylesheet">
+<head>
+<body>
+	<h2>Tropica!</h2>
```

Comment:
Revise all the indentations using the Styleguide provided

Reply:
please throw more light or give an example so that I can review that and others. Thanks so much.